update alerts best practices content (#4430)
getsentry/sentry-docs: src/docs/product/alerts/create-alerts/best-practices.mdx

## Issue Alerts Best Practices

An issue alert is triggered when an individual issue meets some criteria. These criteria (or "triggers") can be based on state-changes or frequency. The best practices that follow cover alerts based on state and frequency changes, as well as reducing noise, and effective routing.

### State-Change Alerts

The following triggers, or “when” conditions, capture state changes in issues:

- A new issue is created
- The issue changes state from resolved to  unresolved or a regression occurs
- The issue changes state from ignored to unresolved — that is, an ignored issue's ignore-until conditions are met

Your first instinct might be to set an alert for every state change. However, this is likely to result in too many alerts if you're running an app with a significant number of users. In particular, regressions will be more common than you expect because Sentry auto-resolves issues after 14 days of silence (configurable), and many issues keep coming back after the 14-day window.

> To deal with this, the **Issues** page includes the **Review List** (in the "For Review" tab), containing only issues that have had a state change in the last seven days. We recommended that you review this list once a day. If you need real-time notifications for particular types of issues, such as those affecting your enterprise customers, you can always create alerts with those filters.

Author: ```suggestion
To deal with this, the **Issues** page includes the [**Review List**](/product/issues/states-triage/#mark-reviewed) (in the "For Review" tab), containing only issues that have had a state change in the last seven days. We recommended that you review this list once a day. If you need real-time notifications for particular types of issues, such as those affecting your enterprise customers, you can always create alerts with those filters.
```

### Frequency-Based Alerts

Below, we describe best practices for setting alerts using the following frequency-based triggers:

- **Number of events in an issue**: This is a very commonly used trigger, but remember that frequency isn't everything: a low-frequency error can be more important than a high-frequency one if it's in a more important part of your app.
- **Number of users affected by an issue**: Sometimes a very small number of users create a lot of errors, so in some cases alerting on users affected can be more important than error frequency. However, remember that not all errors that have user counts in Sentry may be actually user-facing, and vice versa.

> - **Percent of sessions affected by an issue**: Error counts and users affected a) can be hard to reason about, b) require constant manual adjustments as your traffic patterns change, and c) are not well suited to deal with seasonality (for example, fewer errors on the weekend). In such cases, if you've configured your project to capture session data, you can opt for alerting when an issue affects a certain percentage of user sessions.

Author: @adhiraj, I don't understand what is meant by "a) can be hard to reason about". Can you clarify that, please?

Reviewer: I meant "It's difficult to figure out user impact from a count of errors or users-affected."

> - **Percent of sessions affected by an issue**: Error counts and users affected a) can be hard to reason about, b) require constant manual adjustments as your traffic patterns change, and c) are not well suited to deal with seasonality (for example, fewer errors on the weekend). In such cases, if you've configured your project to capture session data, you can opt for alerting when an issue affects a certain percentage of user sessions.

Author: ```suggestion
- **Percent of sessions affected by an issue**: Error counts and users affected require constant manual adjustments as your traffic patterns change and are not well suited to deal with seasonality (for example, fewer errors on the weekend). Also, it can be hard to assess the impact of an issue from error counts or counts of users affected. In such cases, if you've configured your project to capture session data, you can opt for alerting when an issue affects a certain percentage of user sessions.
```
